Makefile

```diff
@@ -18,8 +26,28 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_BUILD) \
                --enable-parser=h264 \
                --enable-parser=hevc
 
+CGO_LDFLAGS := -L$(FFMPEG_PREFIX)/lib
```

Comment:
Why is this duplicated both here & on line 49?

Reply:
Thanks! Fixed. 

Also, did not need to add `$(NDK_ROOT)/toolchains/llvm/prebuilt/$(SOURCE_OS)-x86_64/sysroot/usr/lib` path.